### PR TITLE
Task#1 - Feat: Create README

### LINKED ISSUE #3: Setup enviroment & structure for Practice-one
### OVERVIEW
Setting Practice-one
### Check list
- [x]  Create README.md file
- [x]  Create practice by vite
- [x]  Setup package.json file
- [x]  Create .editorconfig file
- [x]  Install ESlint
- [x]  Add prettierrc 
- [x]  Add .gitignore 
- [x]  Setup structure

### PR DESCRIPTION
**Pull request by TheHienDuong**
- Update readme

### CODE REVIEW

File: practice-one/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# React #Practice-one
+
+## OVERVIEW
+
+* Build the UI of a page with React, creating components from the old HTML/CSS design:
```
Comment (phattruong-agilityio) [inline practice-one/README.md]: from the design

File: practice-one/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    - Prettier is a code formatting tool that automatically formats code based on predefined rules. It supports various programming languages, integrates with popular code editors, and promotes consistent code style. It reduces debates, enhances code readability, and saves time on manual formatting tasks.
+* [Eslint](https://eslint.org/)
+    - ESLint is a popular JavaScript code analysis tool. It identifies and fixes issues, enforces coding conventions, and improves code quality. By applying configurable rules, it catches errors, enforces best practices, and provides real-time feedback. ESLint integrates easily and enhances code quality, readability, and maintainability in JavaScript projects.
+* [Vite](https://vitejs.dev/)
+    - Vite is a lightweight build tool for modern web apps. It supports fast development with instant updates, native ES module handling, and efficient bundling for production. It improves performance and enables quick iteration in JavaScript applications.
+
+## TARGET
+
+* Apply what you have read about the main concepts of ReactJS
+* Can apply ReactJS to build a  website
```
Comment (phattruong-agilityio) [inline practice-one/README.md]: 2 spaces

File: practice-one/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    pnpm install
+    ~~~
+
+* Step 6 After installing the packages:
+    ~~~
+    pnpm start
+    ~~~
+
+* Step 7 Open **http://localhost:5173/src/** in browser
+## Author
```
Comment (phattruong-agilityio) [inline practice-one/README.md:79]: Add empty line

File: practice-one/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    ~~~
+    pnpm install
+    ~~~
+
+* Step 6 After installing the packages:
+    ~~~
+    pnpm start
+    ~~~
+
+* Step 7 Open **http://localhost:5173/src/** in browser
```
Comment (phattruong-agilityio) [inline practice-one/README.md]: Check this link

File: practice-one/README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+* Start date: 28/06/2023
+* Estimate time: 5 days
+* End date: 03/27/2023 (Except Saturday and Sunday)
+* Task management: [Trello](https://trello.com/b/sZ2HEiS3/reactjs-practice-01)
+
+## TECHNICAL
+
+* [HTML5](https://developer.mozilla.org/en-US/docs/Web/HTML)/[CSS3](https://developer.mozilla.org/en-US/docs/Web/CSS)
+  + HTML5 and CSS3 are two core technologies for building web pages. HTML5 provides the structure of the page, while CSS3 provides the visual and aural layout for various devices
```
Comment (phattruong-agilityio) [inline practice-one/README.md]: missing `.`

File: practice-one/README.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+* [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+  + JavaScript is an interpreted programming language used on both the client and server side, allowing for the creation of professional animations on web pages. It is integrated and embedded into HTML to support more dynamic websites
```
Comment (phattruong-agilityio) [inline practice-one/README.md]: missing `.`

File: practice-one/README.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+* [ES6](http://es6-features.org/)
+  + ES6, also known as ECMAScript 2015, is the 6th version of the ECMAScript programming language. It was released in 2015 and brought new syntax and features to make code more modern and readable. Some of these features include arrow functions, template strings, class destruction, and modules
```
Comment (phattruong-agilityio) [inline practice-one/README.md]: missing `.`

File: practice-one/README.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+* [JSON](https://www.w3schools.com/js/js_json_intro.asp)
+  + JSON stands for JavaScript Object Notation, which is a data format that follows a certain rule that most programming languages today can read. JSON is an open standard for data interchange on the web
```
Comment (phattruong-agilityio) [inline practice-one/README.md]: missing `.`

File: practice-one/README.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+* [React](https://react.dev/learn)
+  + React is a JavaScript library for building reusable user interfaces efficiently. It uses a virtual DOM for optimal rendering and updates only what's necessary. With a component-based approach, React simplifies application development and maintenance. It's popular for its simplicity, flexibility, and strong community support.
+
+## DEVELOPMENT TOOLS:
```
Comment (phattruong-agilityio) [inline practice-one/README.md]: Why have `:` here

**Review by phattruong-agilityio — CHANGES_REQUESTED** [review]

File: practice-one/README.md (hunk 1 above)
Comment (TheHienDuong) [reply practice-one/README.md]: Done, I updated

File: practice-one/README.md (hunk 2 above)
Comment (TheHienDuong) [reply practice-one/README.md]: Done, removed space

File: practice-one/README.md (hunk 3 above)
Comment (TheHienDuong) [reply practice-one/README.md:79]: Done! Add empty line 78

File: practice-one/README.md (hunk 4 above)
Comment (TheHienDuong) [reply practice-one/README.md]: Checked and it is correct link when running pnpm dev command

File: practice-one/README.md (hunk 5 above)
Comment (TheHienDuong) [reply practice-one/README.md]: Done

File: practice-one/README.md (hunk 6 above)
Comment (TheHienDuong) [reply practice-one/README.md]: Done

File: practice-one/README.md (hunk 7 above)
Comment (TheHienDuong) [reply practice-one/README.md]: Done

File: practice-one/README.md (hunk 8 above)
Comment (TheHienDuong) [reply practice-one/README.md]: Done

File: practice-one/README.md (hunk 9 above)
Comment (TheHienDuong) [reply practice-one/README.md]: Done. Removed :

**Pull request merged**